standup_notes/__main__.py

```diff
@@ -2,6 +2,7 @@
 import sys
 from datetime import datetime, date, timedelta
 from typing import Callable
+from mattermostdriver import Driver
```

Comment:
You don't need this for this PR, do you?

Reply:
Oh shoot, got aheead of myself :)